external-docs/docs/guides/configuration/code-list-config.md

```diff
@@ -0,0 +1,195 @@
+    "description": "This mixes together concepts from the NUTS and ONS Geographies code lists to solve the NUTS 'England' problem.",
+    "summary": "Mix of NUTS and ONS Geographies concepts.",
+    "creator": "http://dbpedia.org/resource/Open_Knowledge_Foundation",
+    "publisher": "http://dbpedia.org/resource/Open_Knowledge_Foundation",
+    "dataset_issued": "2022-03-31T12:54:30Z",
+    "dataset_modified": "2022-04-01T12:54:30Z",
+    "license": "https://creativecommons.org/licenses/by/4.0/",    
+    "themes": [
+        "http://example.com/themes/theme"
+    ],
+    "keywords": [
+        "NUTS",
+        "ONS Geographies"
+    ],
+    "sort": {
+        "by": "label",
+        "method": "ascending"
+    },
+    "concepts": [
+        {
+            "label": "United Kingdom",
+            "notation": "UK",
+            "same_as": "http://data.europa.eu/nuts/code/UK",
+            "children": [
+                {
+                    "label": "Wales",
+                    "notation": "UKL",
+                    "same_as": "http://data.europa.eu/nuts/code/UKL",
+                    "sort_order": 0
+                },
+                {
+                    "label": "Scotland",
+                    "notation": "UKM",
+                    "same_as": "http://data.europa.eu/nuts/code/UKM"
+                },
+                {
+                    "label": "England",
+                    "notation": "E92000001",
+                    "same_as": "http://statistics.data.gov.uk/id/statistical-geography/E92000001",
+                    "children": [
+                        {
+                            "label": "London",
+                            "notation": "UKI",
+                            "same_as": "http://data.europa.eu/nuts/code/UKI",
+                            "sort_order": 0
+                        },
+                        {
+                            "label": "North East (England)",
+                            "notation": "UKC",
+                            "same_as": "http://data.europa.eu/nuts/code/UKC",
+                            "sort_order": 2
+                        },
+                        {
+                            "label": "North West (England)",
+                            "notation": "UKD",
+                            "same_as": "http://data.europa.eu/nuts/code/UKD",
+                            "sort_order": 1
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+```
+
+
+## Defining an in-line code list
+
+This approach is recommended for defining simple code lists (e.g. code lists with a small number of concepts or simple hierarchy).
+
+As shown in the below example, an in-line code list is defined within the `qube-config.json` using the same fields discussed above. Moreover, as shown in the example, one can choose to define a `code-list-config.json` for a one dimension and define an in-line code list for another dimension.
+
+```json
```

Comment:
Is this mixed geography code list a good example here? I think a simpler example should be a scenario like "Flow" with concepts "Imports" "Exports" "Other". Covering why you would want to mix concepts could be an effective how-to scenario, since we need to explain why someone wants to do it.

Reply:
Yeah, I think I agree, if we can keep the code list as simple as possible we're probably going to make it easier for the user to understand the underlying idea we're trying to convey.

Reply:
Thanks @canwaf @robons . I will update accordingly.